tasks/ChainPublisher.hpp

```diff
@@ -21,22 +20,8 @@ class ChainPublisher : public ChainPublisherBase
 {
     friend class ChainPublisherBase;
 protected:
-    std::vector<KDL::Chain> chains_;
-    std::vector<std::string> chain_names_;
-    std::vector<KDL::ChainFkSolverPos_recursive*> pos_solvers_;
-    std::vector<KDL::Frame> kdl_frames_;
-    std::vector<base::samples::RigidBodyState> bt_frames_;
-    std::vector<KDL::JntArray> joint_arrays_;
-    std::vector<std::vector<std::string> >involved_active_joints_;
-    base::samples::Joints joint_state_;
-    KDL::Tree tree_;
-    int n_defined_chains_;
-    std::vector<RTT::OutputPort<base::samples::RigidBodyState>*> out_ports_;
-
-    void clear_and_resize_vectors();
-    bool unpack_joints(const base::samples::Joints& joint_state,
-                       const std::vector<std::string>& involved_joints,
-                       KDL::JntArray& joint_array);
+     base::samples::Joints joint_state;
+     std::vector<RTT::OutputPort<base::samples::RigidBodyState>*> out_ports_;
```

Comment:
I guess a member variable for the new `ChainTransformationCalculator` is missing here

Reply:
yeah, i'll will add that